lib/hubstats/events_handler.rb

```diff
@@ -55,15 +55,13 @@ def comment_processor(payload, kind)
     #
     # Returns - nothing, but updates or makes the team
     def team_processor(payload)
-      team = payload[:team]
-      team[:action] = payload[:action]
-      team[:current_user] = payload[:member]
+      team = payload[:event][:team]
```

Comment:
I don't see a `event` attribute in this payload. https://developer.github.com/v3/activity/events/types/#membershipevent

Am I missing something here?